python/src/vmaf/core/noref_feature_extractor.py

```diff
@@ -4,30 +4,30 @@
 from scipy.special._ufuncs import gamma
 from skimage.util import view_as_windows
 from PIL import Image
+import numpy as np
+import copy
+from scipy import ndimage
 
 from vmaf.core.executor import NorefExecutorMixin
 
 __copyright__ = "Copyright 2016-2018, Netflix, Inc."
```

Comment:
not directly related to this PR, but maybe in another PR we can replace the copyright with latest dates, e.g. 2016-2020

Reply:
Ok. we can do that (in a different PR).